NEWS

```diff
@@ -12,18 +12,18 @@
 ### Bugfixes:
 
 ## 1.18.0-rc3 (December 18, 2024)
-### Feature
+### Features:
 #### UCP
  * Enabled using CUDA staging buffers for pipeline protocols by default
  * Added endpoint reconfiguration support for non-reused p2p scenarios
- * Enabled non-cacheable memory domains, activated for GDR
+ * Enabled non-cacheable memory domains, activated for GDR copy
```

Comment:
GDR copy -> gdr_copy
 lets make the naming consistent

Reply:
fixed all